backend/app.py

```diff
@@ -25,11 +32,18 @@
     allow_headers=['*']
 )
 
-# jq docs: https://stedolan.github.io/jq/manual/
-# jq python api doc: https://github.com/mwilliamson/jq.py
-CSETS_JSON = f'{PROJECT_DIR}/termhub-csets/temp/objects/OMOPConceptSet/latest.json'
+
+# Utils
+def jq_wrapper(query: str):
```

Comment:
@Sigfried Figured it would be good to re-use this block of code which we'll probably need each time we want to call JQ from the shel.

It was more difficult to get the shell to work with JQ in this instance. It seemed to have something to do with `'` present in the `query` that I was trying to call. I tried escaping and a bunch of other stuff, I couldn't get my typical, simple `subprocess.call(query.split(' '))` one-liner to work. I ended up needing these several lines to open up a process, grab the output from that process, and decode the bytes.